packages/jest-snapshot/src/utils.js

```diff
@@ -69,6 +69,8 @@ const validateSnapshotVersion = (snapshotContents: string) => {
       `Received: v${version}`
     );
   }
+
+  return true;
```

Comment:
could we just "return null" otherwise? That way we don't need to do a typecheck.